docs/openapi.yaml

```diff
@@ -706,8 +706,8 @@ paths:
   '/ar-io/admin/queue-tx':
     post:
       tags: [Admin]
-      summary: Prioritize a transaction in the queue.
-      description: Stage a specific TX ID as priority for your Gateway to locate and index.
+      summary: Prioritize a transaction or bundle in the queue.
+      description: Stage a specific TX ID as priority for your Gateway to locate and index. This can be any L1 transaction, or bundle if your Gateway is configured to unbundle and index.
```

Comment:
I worry this will be confusing since you can't actually queue a nested bundle using this endpoint. Maybe we should keep the original wording, but add something like "this will trigger bundle processing if the transaction the TX is a bundle".